sdk/ml/azure-ai-ml/azure/ai/ml/_schema/_deployment/batch/batch_job.py

```diff
@@ -52,15 +52,12 @@ class BatchJobSchema(PathAwareSchema):
     @post_load
     def make(self, data: Any, **kwargs: Any) -> Any:
         if data.get(EndpointYamlFields.BATCH_JOB_INPUT_DATA, None):
-            input_data = data[EndpointYamlFields.BATCH_JOB_INPUT_DATA]["input_data"]
-            if isinstance(input_data, Input):
-                if input_data.type == AssetTypes.URI_FILE:
-                    data[EndpointYamlFields.BATCH_JOB_INPUT_DATA] = {"uriFile": UriFileJobInput(uri=input_data.path)}
-                elif input_data.type == AssetTypes.URI_FOLDER:
-                    data[EndpointYamlFields.BATCH_JOB_INPUT_DATA] = {
-                        "uriFolder": UriFolderJobInput(uri=input_data.path)
-                    }
-
+            for key, input_data in data[EndpointYamlFields.BATCH_JOB_INPUT_DATA].items():
```

Comment:
Will this throw if the key is not in the dict?

Reply:
Key will be in the dictionary since we are iterating over the dictionary items.